src/wallet/spend.cpp

```diff
@@ -301,19 +296,13 @@ CoinsResult AvailableCoins(const CWallet& wallet,
 
 CoinsResult AvailableCoinsListUnspent(const CWallet& wallet, const CCoinControl* coinControl, const CAmount& nMinimumAmount, const CAmount& nMaximumAmount, const CAmount& nMinimumSumAmount, const uint64_t nMaximumCount, bool include_immature_coinbase)
```

Comment:
427548d8df0b2786ee360f443bb07adc264773bc
maybe you could wrap these args as well?

Reply:
I didn't do it because of the `only_spendable` arg that must be true for this method (otherwise shouldn't be called "AvailableCoins**ListUnspent**").
But.. could also pass the struct and force set the `only_spendable` flag internally too.

Reply:
@furszy if you do decide to do that, maybe by using the struct here as well
https://github.com/bitcoin/bitcoin/blob/0bac04b7585017508425b21d402fd3c52e6a13d5/src/wallet/rpc/coins.cpp#L593-L596
it creates a good opportunity (as you'll now touch all the existing code that uses them) to [rename the variables to snake_case](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c)?

Reply:
Pushed both changes.

Squashed the `AvailableCoinsListUnspent` changes inside  56c7d7d and added an scripted-diff commit 1022c18 at the end for the snake_case rename to minimize the review burden.